bin/helm-deploy

```diff
@@ -17,6 +17,14 @@ if [[ ! $(kubectl get namespace $NAMESPACE) ]]; then
   kubectl create namespace $NAMESPACE;
 fi
 
+# Set tiller_namespace if it isn't already set.
+# Allows us to use helm even if it's not installed
+# in kube-system.
+if [ -z "$TILLER_NAMESPACE" ]; then
+  TILLER_NAMESPACE="$(kubectl get deploy -o wide --all-namespaces | grep tiller-deploy | tail -n 1 | cut -d ' ' -f 1 | tr -d ' ')"
```

Comment:
Instead of the first grep, we could use a label selector `kubectl get deploy --all-namespaces -l app=helm,name=tiller`.